js/app.js

```diff
@@ -133,15 +133,51 @@ function randomizeGame() {
   }
 }
 
+function getIconPath(faction) {
+  var iconName = "";
+  switch (faction) {
```

Comment:
I think the iconNames should go into data.js: https://github.com/haswellr/AllRandomAllRoot/blob/main/js/data.js#L1. Then you can avoid the big switch and we've got all faction-specific data in one place.

Reply:
Yea I wasn't sure if I wanted to associate the data for the game selection with the front end decision, but I think it's fine the way you suggest. I moved the iconName to properties on the data objects. Lmk